CostumersAPI/CustomExtensions/StringExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace CostumersAPI.CustomExtensions
+{
+    public static class StringExtensions
+    {
+        public static bool IsValidCPF(this string document)
+        {
+            var documentValidator = new Regex(@"^\d{3}\.?\d{3}\.?\d{3}\-?\d{2}$");
+            if (document == null) return false;
+            if (!documentValidator.IsMatch(document)) return false;
```

Comment:
```suggestion
        public static bool IsValidCPF(this string document)
        {
            if (document == null) return false;
            var documentValidator = new Regex(@"^\d{3}\.?\d{3}\.?\d{3}\-?\d{2}$");
            if (!documentValidator.IsMatch(document)) return false;
```
Sempre que possível, tentar trazer as possibilidades de insucesso antes, para que não compile código que não foi utilizado.